README.md

```diff
@@ -32,14 +32,14 @@ jobs:
   auto-label:
     runs-on: ubuntu-latest
     steps:
-      - uses: prince-chrismc/label-merge-conflicts-action@v1
+      - uses: prince-chrismc/label-merge-conflicts-action@v1 # Replace this with the latest version
         with:
           conflict_label_name: "has conflict"
           github_token: ${{ secrets.GITHUB_TOKEN }}
           # These are optional incase you need to adjust for the limitations described below
           max_retries: 5
           wait_ms: 15000
-          detect_merge_changes: false # or true to handle as conflicts
+          mergeable_only: false # or true to unlabel as soon as all the merge requirements are fulfilled
```

Comment:
If I understand your explanation so far....

I do not think including branch permissions is something that match my use case, I strictly only care about modifications to the files.

This would falsely label something as a "conflict" say if the CI or end-to-end test failed and it was required by a branch permission?